crates/editor/src/editor.rs

```diff
@@ -16493,9 +16532,12 @@ impl Editor {
                     target_buffer.anchor_after(target_start)
                         ..target_buffer.anchor_before(target_end)
                 })?;
-                Location {
-                    buffer: target_buffer_handle,
-                    range,
+                LocationLink {
+                    origin: None,
```

Comment:
I don't know enough about this code to know if we can get an `origin` for this jump